src/markdown_exec/__init__.py

```diff
@@ -24,10 +24,13 @@
 from markdown_exec.formatters.python import _format_python
 from markdown_exec.formatters.sh import _format_sh
 from markdown_exec.formatters.tree import _format_tree
+from markdown_exec.formatters.typescript import _format_typescript
 
 __all__: list[str] = ["formatter", "validator"]
 
-MARKDOWN_EXEC_AUTO = [lang.strip() for lang in os.getenv("MARKDOWN_EXEC_AUTO", "").split(",")]
+MARKDOWN_EXEC_AUTO = [
+    lang.strip() for lang in os.getenv("MARKDOWN_EXEC_AUTO", "").split(",")
+]
```

Comment:
style: Early note to recommend disabling auto-formatting in your IDE/editor, or have it respect our config in `config/ruff.toml` :slightly_smiling_face:

Reply:
will do! good to get ahead of this, but also I haven't thought about those final "polishing" steps at all for this yet 😄